### PR TITLE
Remove `bycol` ops in vertical diffusion function

### PR DESCRIPTION
**Pull request by akshaysridhar**
Removes one of the pending `bycolumn` operations from the vertical diffusion Frierson function. `colidx` is still used in some tendency calculations (gravity wave, external forcing)

### CODE REVIEW

**Comment by akshaysridhar** [comment]
I'll run a `longrun` sweep, but since this is a test of particular interest, here are the results for the `longrun_aquaplanet_clearsky_1M` job_id, run to the default longrun simulation end time which is 120 days. 
[summary.pdf](https://github.com/user-attachments/files/15930828/summary.pdf)
[summary_2D.pdf](https://github.com/user-attachments/files/15930827/summary_2D.pdf)

cc @trontrytel

**Comment by trontrytel** [comment]
I think those look reasonable!

File: config/longrun_configs/longrun_aquaplanet_clearsky_1M.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ z_max: 55000.0
 rayleigh_sponge: true
 moist: "equil"
 precip_model: "1M"
+hyperdiffusion: "CAM_SE"
```
Comment (szy21) [inline config/longrun_configs/longrun_aquaplanet_clearsky_1M.yml:11]: This should not be needed as CAM_SE is the default.

**Review by szy21 — APPROVED** [review]

**Comment by akshaysridhar** [comment]
@szy21 I added a version update change as well to this PR so we can release this before the longruns get started.

**Comment by akshaysridhar** [comment]
(version update removed)

**Comment by szy21** [comment]
The plots in summary look strange to me. Specific humidity seems uniform, and there seem to be no rain, or cloud ice, or liquid water. They don't seem to be consistent with the plots in summary_2D. Could you double check?

**Pull request merged**